flutter/lib/sentry_flutter.dart

```diff
@@ -9,4 +13,97 @@ mixin SentryFlutter {
     final String version = await _channel.invokeMethod('getPlatformVersion');
     return version;
   }
+
+  static void _flutterErrorIntegration(Hub hub, SentryOptions options) {
+    final defaultOnError = FlutterError.onError;
+
+    FlutterError.onError = (FlutterErrorDetails errorDetails) async {
+      options.logger(
+          SentryLevel.debug, 'Capture from onError ${errorDetails.exception}');
+
+      // TODO: create mechanism
+
+      await hub.captureException(
+        errorDetails.exception,
+        stackTrace: errorDetails.stack,
+      );
+
+      // call original handler
+      if (defaultOnError != null) {
+        defaultOnError(errorDetails);
+      }
+
+      // do we need this?
+      // print('Capture from FlutterError ${details.exception}');
+      // Zone.current.handleUncaughtError(details.exception, details.stack);
```

Comment:
I believe for as long as we forward the call to the original handler (line 32) we're good here.